node/src/chain_spec.rs

```diff
@@ -178,24 +230,40 @@ fn testnet_genesis(
         },
         balances: BalancesConfig {
             // Configure endowed accounts with initial balance of 1 << 60.
-            balances: endowed_accounts.iter().cloned().map(|k| (k, 1 << 60)).collect(),
-        },
-        aura: AuraConfig {
-            authorities: initial_authorities.iter().map(|x| (x.0.clone())).collect(),
-        },
-        grandpa: GrandpaConfig {
-            authorities: initial_authorities.iter().map(|x| (x.1.clone(), 1)).collect(),
+            balances: endowed_accounts.iter().cloned().map(|k| (k, ENDOWMENT)).collect(),
         },
+        // This has been initialized from the session config
+        aura: Default::default(),
+        // This has been initialized from the session config
+        grandpa: Default::default(),
         sudo: SudoConfig {
             // Assign network admin rights.
             key: root_key,
         },
         utxo: UtxoConfig {
             genesis_utxos: vec![genesis],
+            // The # of validators set should also be the same here.
+            // Currently forcing only 1 INITIAL AUTHORITY (Alice), hence only 1 locked-genesis.
+            // This means only ALICE has staked her utxo.
+            locked_utxos: vec![locked_genesis],
+            extra_mlt_coins: 50_000 as pallet_utxo::Value,
+            initial_reward_amount: 100 as pallet_utxo::Value,
```

Comment:
What's the difference between initial reward and extra mlt?

Reply:
`initial_reward_amount` is the reward  for each block:

> "The first year each block includes 100 MLT tokens, the second year the reward is decreased to 75 MLT tokens and so on until 200 millions new MLT tokens are generated."

-------

The `extra_mlt_coins` is the `200_000_000 MLT` . That's part of my _todo_, 
> We have 200,000,000 to add in over time. Some of that will be from a faucet. Some of that could be just every block adding some mlt to the block reward

-------
for every block reward, it deducts from the `extra_mlt_coins`.

Reply:
Gotcha. I understand now